src/assets/translations/en.translations.json

```diff
@@ -36,9 +36,9 @@
   "modal.text.7": " Yes",
   "modal.text.8": " No",
   "modal.text.9": "Select your dietary type",
-  "modal.text.10": "Regular menu",
```

Comment:
I don't want you to change the translation string but the `values` instead you have to revert this change please

Reply:
I reverted this as well

Reply:
I added the selector and I changed the button style in the form